lab_share_lib/constants.py

```diff
@@ -1,2 +1,9 @@
+from typing import Final
+
 RABBITMQ_HEADER_KEY_SUBJECT = "subject"
 RABBITMQ_HEADER_KEY_VERSION = "version"
+
+###
+# Logger names
+###
+LOGGER_NAME_RABBIT_MESSAGES: Final[str] = "rabbit_messages"
```

Comment:
Would it be worth putting the `: Final[str]` type hints on the other keys above as well?